src/containers/App.js

```diff
@@ -34,7 +35,8 @@ class IntlTelInput extends Component {
   render() {
     return (
       <Provider store={store}>
-        <IntlTelInputApp value={this.props.value}
+        <IntlTelInputApp id={this.props.id}
```

Comment:
Generate a `uuid` here, then the prop is not necessary .. see https://github.com/shtylman/node-uuid